<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QStaticPictureDlg</class>
 <widget class="QDialog" name="QStaticPictureDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>522</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GIF显示器</string>
  </property>
  <widget class="QPushButton" name="m_helpBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>帮助</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_okBtn">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>88</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>预览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_cancelBtn">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>490</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>131</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>控件类型</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 85, 0);</string>
    </property>
    <property name="text">
     <string>GIF显示器</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>41</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="maximumSize">
        <size>
         <width>30</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox">
       <property name="minimum">
        <number>-1</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGraphicsView" name="MainView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>112</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>591</width>
     <height>471</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>基本</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>551</width>
       <height>111</height>
      </rect>
     </property>
     <property name="title">
      <string>图片属性</string>
     </property>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>选择图片：</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_runstopbtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>运行测试</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_pathbtn">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>20</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/standardtool/images/standardtool/open.ico</normaloff>:/standardtool/images/standardtool/open.ico</iconset>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>70</y>
        <width>108</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string> （0表示一直执行）</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="m_nRCountspBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>191</width>
        <height>20</height>
       </rect>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>2147483647</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>60</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>执行次数：</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>551</width>
       <height>151</height>
      </rect>
     </property>
     <property name="title">
      <string>控制</string>
     </property>
     <widget class="QRadioButton" name="available_0">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>115</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>0</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>69</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>控制位：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="ControlBit">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QRadioButton" name="available_1">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>115</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>1</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="IfBitControl">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>受位控制</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>70</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="../../resourse.qrc">
        <normaloff>:/addressintput/images/address.bmp</normaloff>:/addressintput/images/address.bmp</iconset>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>115</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>有效状态：</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_tips">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>131</width>
     <height>231</height>
    </rect>
   </property>
   <property name="title">
    <string>提示</string>
   </property>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>201</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(60, 63, 255);</string>
    </property>
    <property name="text">
     <string>GIF显示器用于显示指定位置的GIF动画，使用GIF显示器可以显示GIF格式的动画图片</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../../resourse.qrc"/>
 </resources>
 <connections/>
</ui>
